spec/feature/authenticated_traveler_spec.rb

```diff
@@ -83,16 +83,32 @@
     expect(page).to have_content("You must be a host to see your guests")
     expect(current_path).to eq(root_path)
   end
+end
 
-  xit "can add and remove properties from a wish list" do
-    login(user)
+describe "traveler permissions", type: :feature, js: true do
+  let!(:user) { create(:user) }
+  let!(:property) { create(:property) }
+
+  it "can add and remove properties from a wish list" do
+    visit login_path
+    fill_in("email_address", with: user.email_address)
+    fill_in("password", with: user.password)
+    find_button("Login").click
+
+    visit wishlist_path
+    expect(page).to have_content("YOU DON'T HAVE ANYTHING ON YOUR WISHLIST")
```

Comment:
Yeah, that's how the styling is for all of the headers, so that's what it's looking for. I tried testing it with less shouty spelling and it didn't work.